### PR TITLE
feat: Adding API v2 Bigquery export samples

### PR DESCRIPTION
**Pull request by rafaelMurata**
## Description

b/335222583

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist

- [ ] I have followed [Sample Format Guide](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/SAMPLE_FORMAT.md)
- [ ] `pom.xml` parent set to latest `shared-configuration`
- [ ] Appropriate changes to README are included in PR
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] **Tests** pass:   `mvn clean verify` **required**
- [ ] **Lint**  passes: `mvn -P lint checkstyle:check` **required**
- [ ] **Static Analysis**:  `mvn -P lint clean compile pmd:cpd-check spotbugs:check` **advisory only**
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample 
- [ ] Please **merge** this PR for me once it is approved

### CODE REVIEW

File: security-command-center/snippets/src/main/java/vtwo/bigquery/DeleteBigQueryExport.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    deleteBigQueryExport(organizationId, location, bigQueryExportId);
+  }
+
+  // Delete an existing BigQuery export.
+  public static void deleteBigQueryExport(String organizationId, String location,
+      String bigQueryExportId)
+      throws IOException {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the "close" method on the client to safely clean up any remaining background resources.
```
Comment (Sita04) [inline security-command-center/snippets/src/main/java/vtwo/bigquery/DeleteBigQueryExport.java:48]:
```suggestion
    // once, and can be reused for multiple requests.
```

File: security-command-center/snippets/src/main/java/vtwo/bigquery/GetBigQueryExport.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+    getBigQueryExport(organizationId, location, bigQueryExportId);
+  }
+
+  // Retrieve an existing BigQuery export.
+  public static void getBigQueryExport(String organizationId, String location,
+      String bigQueryExportId) throws IOException {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the "close" method on the client to safely clean up any remaining background resources.
```
Comment (Sita04) [inline security-command-center/snippets/src/main/java/vtwo/bigquery/GetBigQueryExport.java:48]:
```suggestion
    // once, and can be reused for multiple requests.
```

File: security-command-center/snippets/src/main/java/vtwo/bigquery/ListBigQueryExports.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+    listBigQueryExports(organizationId, location);
+  }
+
+  // List BigQuery exports in the given parent.
+  public static void listBigQueryExports(String organizationId, String location)
+      throws IOException {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the "close" method on the client to safely clean up any remaining background resources.
```
Comment (Sita04) [inline security-command-center/snippets/src/main/java/vtwo/bigquery/ListBigQueryExports.java:46]:
```suggestion
    // once, and can be reused for multiple requests.
```

File: security-command-center/snippets/src/main/java/vtwo/bigquery/UpdateBigQueryExport.java [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+    updateBigQueryExport(organizationId, location, filter, bigQueryExportId);
+  }
+
+  // Updates an existing BigQuery export.
+  public static void updateBigQueryExport(String organizationId, String location, String filter,
+      String bigQueryExportId)
+      throws IOException {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the "close" method on the client to safely clean up any remaining background resources.
```
Comment (Sita04) [inline security-command-center/snippets/src/main/java/vtwo/bigquery/UpdateBigQueryExport.java:58]:
```suggestion
    // once, and can be reused for multiple requests.
```

File: security-command-center/snippets/src/test/java/vtwo/BigQueryExportIT.java [hunk 5]
```diff
@@ -0,0 +1,224 @@
...
+          .build();
+      // Mock the createBigQueryExport method to return the expected response.
+      when(client.createBigQueryExport(any())).thenReturn(expectedExport);
+
+      // Call the createBigQueryExport method.
+      CreateBigQueryExport.createBigQueryExport(ORGANIZATION_ID, LOCATION, PROJECT_ID, filter,
+          BQ_DATASET_NAME, BQ_EXPORT_ID);
+
+      // Verify that the createBigQueryExport method was called with the expected request.
+      verify(client).createBigQueryExport(expectedRequest);
```
Comment (Sita04) [inline security-command-center/snippets/src/test/java/vtwo/BigQueryExportIT.java:90]: Assert if the response from the snippet is as expected.
Applies to all testcases.

File: security-command-center/snippets/src/test/java/vtwo/BigQueryExportIT.java [hunk 6]
```diff
@@ -0,0 +1,224 @@
...
+  private static final String BQ_DATASET_NAME = "test-dataset-id";
+  private static final String BQ_EXPORT_ID = "test-export-id";
+
+  @Test
+  public void testCreateBigQueryExport() throws IOException {
+    // Mock SecurityCenterClient.
+    SecurityCenterClient client = mock(SecurityCenterClient.class);
+    try (MockedStatic<SecurityCenterClient> clientMock = Mockito.mockStatic(
+        SecurityCenterClient.class)) {
+      clientMock.when(SecurityCenterClient::create).thenReturn(client);
```
Comment (Sita04) [inline security-command-center/snippets/src/test/java/vtwo/BigQueryExportIT.java:63]: Isn't the MockedStatic class redundant here? Applies to all testcases..

**Review by Sita04 — CHANGES_REQUESTED** [review]

File: security-command-center/snippets/src/test/java/vtwo/BigQueryExportIT.java (hunk 5 above)
Comment (rafaelMurata) [reply security-command-center/snippets/src/test/java/vtwo/BigQueryExportIT.java]: done

File: security-command-center/snippets/src/test/java/vtwo/BigQueryExportIT.java (hunk 6 above)
Comment (rafaelMurata) [reply security-command-center/snippets/src/test/java/vtwo/BigQueryExportIT.java:88]: at first I thought the same, but kokoro doesn't accept any other way. But I don't think it's much more different from the documentation's suggestion.
https://javadoc.io/doc/org.mockito/mockito-core/latest/org/mockito/Mockito.html#48

**Review by Sita04 — APPROVED** [review]

**Pull request merged**